prod.Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM eclipse-temurin:11.0.15_10-jdk-alpine AS stage1
+# syntax=docker/dockerfile:1
+FROM --platform=$BUILDPLATFORM eclipse-temurin:11.0.15_10-jdk-alpine AS stage1
```

Comment:
Given that everywhere else you are building images from scratch, would it make sense to build here one too for consistency? Or there are certain benefits in using eclipse-temurin here?

Reply:
I'm not entirely sure, but given that this is the production image, we do want it to be as secure as possible.  I think it's reasonable to have a dependance on the package management system in our development images, given that those are just running locally, but there's a bit of an extra layer of confidence, security, and validation in using officially vetted docker images for our production instance.  The things they do for the official images are in this dockerfile for jdk: https://github.com/adoptium/containers/blob/main/11/jdk/alpine/Dockerfile.releases.full and this for jre: https://github.com/adoptium/containers/blob/main/11/jre/alpine/Dockerfile.releases.full 

They take several steps to be more secure, like checksums, and pinning specific versions of java